README.md

```diff
@@ -47,43 +67,132 @@ $ npm run start:dev
 $ npm run start:prod
 ```
 
+
+### Running via docker
+To execute the build (first exection):
+```bash
+./bin/runenv.sh --build
+```
+Execute the follow command:
+```bash
+./bin/runenv.sh
+```
+### Boot extra resources
+If the project has SQS Queues or others resources you can boot after the environment is ready.
+
+Execute the follow command:
+```bash
+./bin/boot.sh
+```
+
+## Samples
+See the project samples in this folder [here](samples).
```

Comment:
To sound more natural: 

"Check out the project samples at the [samples](samples) directory"